utilize_eye_in_hand_calibration.py

```diff
@@ -0,0 +1,54 @@
+"""
+Utilize the result of eye-in-hand calibration to transform (picking) point
+coordinates from the camera frame to the robot base frame.
+"""
+
+import numpy as np
+import cv2
+
+np.set_printoptions(precision=2)
+
+
+def read_transform(file_name):
+    """Read transformation matrix from a YAML file.
+
+    Args:
+        file_name: Path to the YAML file.
+
+    Returns:
+        transform: Transformation matrix.
+
+    """
+
+    file_storage = cv2.FileStorage(file_name, cv2.FILE_STORAGE_READ)
```

Comment:
Does `cv2.FileStorage` support `pathlib.Path`?

Reply:
It doesn't seem so. The only documentation I found is for c++ and there it only supports string:
https://docs.opencv.org/3.4/da/d56/classcv_1_1FileStorage.html#a29bcb12ede10d87a0fcbf89a88b092af
String is also the only option I found for python FileStorage

Reply:
It is up to you if you would first create a `Path` instance first and then call `str` on it, or simply use raw strings. If you think it would be easier for customers to understand, then leave it be.

Reply:
What is the benefit of using the path class?

Reply:
It makes it clear that what you are looking at is a path, and not some other random string.

Reply:
done